cargo-ledger/src/main.rs

```diff
@@ -117,9 +132,10 @@ fn main() {
         MainCommand::Build {
             device: d,
             load: a,
+            standard_build_tree: s,
             remaining_args: r,
         } => {
-            build_app(d, a, cli.use_prebuilt, cli.hex_next_to_json, r);
+            build_app(d, a, s, cli.use_prebuilt, cli.hex_next_to_json, r);
```

Comment:
Are `use_prebuilt` and `hex_next_to_json`flags really needed 🤔 ?

Reply:
I need to check where we actually use those, I think it's in the deployment CI, maybe we can get rid of them.

Reply:
This is used (and contributed to) by Obsidian Systems